lib/tomo/plugin/rbenv/tasks.rb

```diff
@@ -60,6 +60,8 @@ def extract_ruby_ver_from_version_file
       version = remote.capture("cat", path, raise_on_error: false).strip
       return version unless version.empty?
 
+      return RUBY_VERSION.inspect if dry_run?
```

Comment:
`RUBY_VERSION` is already a string, so `.inspect` is not needed. E.g., instead of this method returning `"2.7.2"`, it causes the return value to be `"\"2.7.2\""`, which I don't think is what we want.

```suggestion
      return RUBY_VERSION if dry_run?
```